HLTrigger/Configuration/python/HLT_75e33/modules/ticlTrackstersMerge_cfi.py

```diff
@@ -1,6 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-ticlTrackstersMerge = cms.EDProducer("TrackstersMergeProducer",
+ticlTrackstersMerge = cms.EDProducer("TrackstersMergeProducerV3",
```

Comment:
Does this mean HLT Phase-2 is running an old version of TICL? When will that version be removed? Is there a plan to update HLT to use TICL V4? @trtomei ?

Reply:
Hi @Martin-Grunewald , yes. 
As I said during the HLT upgrade meeting, the plan is to switch HLT to TICLv4, but this might need some work from POGs to adjust the way they are consuming tracksters.

Reply:
@felicepantaleo 
OK, thanks!